hail/python/hail/experimental/table_ndarray_utils.py

```diff
@@ -18,8 +18,11 @@ def mt_to_table_of_ndarray(entry_expr, block_size=16, return_checkpointed_table_
 
     def get_even_partitioning(ht, partition_size, total_num_rows):
         ht = ht.select().add_index("_even_partitioning_index")
+        info("Added index")
```

Comment:
looks like another commit snuck in?

Reply:
Ugh yes, let me fix.

Reply:
Accidentally forked off a debugging branch, rebased it away.